app/loading-user-data.tsx

```diff
@@ -14,12 +15,17 @@ export default function LoadingScreen() {
   const { data: profilePicture, status: profilePictureStatus } = useProfilePicture(
     userDetails?.attachmentId,
   );
+  const { data: userGroups, status: userGroupsStatus } = useGroups();
 
   const { setUserData } = useContext(GlobalContext);
 
   useEffect(() => {
-    if (userDetailsStatus === 'success' && profilePictureStatus === 'success') {
-      setUserData({ userDetails, profilePicture });
+    if (
+      userDetailsStatus === 'success' &&
+      profilePictureStatus === 'success' &&
+      userGroupsStatus == 'success'
+    ) {
+      setUserData({ currentGroupId: userGroups?.groups[0]?.groupId, userDetails, profilePicture });
```

Comment:
why on the first index of the groups?

Reply:
when the user is logged application goes to /groups and there is select list with one group already selected. So choice of 'currentGroupId' should be made without use ingeration so i simply set it to the first group ruturned in useGroups

Reply:
Ok, but for future I think we must to store group that we have before logout in the memory